builtin/plugins/chainconfig/chainconfig.go

```diff
@@ -350,6 +368,119 @@ func EnableFeatures(ctx contract.Context, blockHeight, buildNumber uint64) ([]*F
 	return enabledFeatures, nil
 }
 
+// UpdateConfig returns a list of settings that are supported by majority of validators
+func UpdateConfig(ctx contract.Context, buildNumber uint64) ([]*Setting, error) {
+	params, err := getParams(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	settingsRange := ctx.Range([]byte(settingPrefix))
+	settings := make([]*Setting, 0)
```

Comment:
These aren't really settings, they're changes to the config, or actions... so that's probably what we should call these. Actions only exist until they're applied to the on-chain config.

Reply:
Done